tests/kernel/update_kernel.pm

```diff
@@ -473,6 +473,32 @@ sub boot_to_console {
       unless is_sle('<12');
 }
 
+sub install_requirements {
+    my @requirements = qw(
+      rasdaemon
+      libnvme1
+      nvme-cli
+      rdma-core
+      rdma-ndd
+      librdmacm1
+      blktrace
+      bpftrace
+      bcc-tools
+      libbcc0
+      tcpdump
+      kdump
+      crash
+      makedumpfile
+      nfs-client
+      nfs-kernel-server
+      open-iscsi
+      multipath-tools
+      liburing2
+    );
+
+    zypper_call("in @requirements");
```

Comment:
We'll probably use the same flavor for Micro 6.2 and later SLES transactional tests where `zypper_call()` will break. Please use `install_package()` instead.

Reply:
done,  VR updated